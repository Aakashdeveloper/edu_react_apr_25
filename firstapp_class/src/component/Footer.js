import React from 'react';

function Footer(props){
    console.log(props)

    //deststructre
    const {month,year} = props
    return(
        <footer>
            <hr/>
            <center>
            {/* <h3>&copy; DeveloperFunnel {props.year} {props.month}</h3> */}
            <h3>&copy; DeveloperFunnel {month} {year}</h3>
            </center>
            
        </footer>
      
    )
}

export default Footer;