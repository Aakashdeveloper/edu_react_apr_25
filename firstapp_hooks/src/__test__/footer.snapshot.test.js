import React from 'react';
import Footer from '../component/Footer';
import {render} from '@testing-library/react';

describe('Snapshot',() => {
    test('Testing Footer',() => {
        const {asFragment} = render(<Footer/>)
        expect(asFragment()).toMatchSnapshot()
    })
    
})