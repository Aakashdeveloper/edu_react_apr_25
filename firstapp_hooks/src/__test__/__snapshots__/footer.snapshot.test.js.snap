// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot Testing Footer 1`] = `
<DocumentFragment>
  <footer>
    <hr />
    <center>
      <h3>
        Â© DeveloperFunnel  
      </h3>
    </center>
  </footer>
</DocumentFragment>
`;
